conf/settings.json

```diff
@@ -46,6 +46,8 @@
     "rpc-authentication-required": false, 
     "rpc-bind-address": "127.0.0.1", 
     "rpc-enabled": true, 
+    "rpc-host-whitelist": "", 
+    "rpc-host-whitelist-enabled": true, 
```

Comment:
I don't understand, why set it to true when in #47 the fix proposed is to set it to false? Or maybe add \_\_DOMAIN\_\_ in the line above?

Reply:
Looks like the fix, https://github.com/YunoHost-Apps/transmission_ynh/pull/48 was wrong, and no one never noticed it...

Reply:
Both with false or true, it works on my server.